tdx-tdcall/src/tdx.rs

```diff
@@ -26,6 +26,7 @@ const TDCALL_TDGETVEINFO: u64 = 3;
 const TDCALL_TDACCEPTPAGE: u64 = 6;
 
 const TDVMCALL_MAPGPA: u64 = 0x10001;
+const TDVMCALL_GETINFO: u64 = 0x10000;
```

Comment:
Please 
1) rename it TDVMCALL_GETTDVMCALLINFO
2) move it before TDVMCALL_MAPGPA